ui/src/logs/containers/LogsPage.tsx

```diff
@@ -36,6 +37,7 @@ interface Props {
   changeZoomAsync: (timeRange: TimeRange) => void
   executeQueriesAsync: () => void
   setSearchTermAsync: (searchTerm: string) => void
+  fetchMoreAsync: (queryTimeEnd: string, lastTime: number) => Promise<void>
```

Comment:
Fetch more what?  Nit here, but if this is ever used in another component might want to be more specific